src/lib/components/Header/Header.svelte

```diff
@@ -19,15 +25,9 @@
 
 <!-- TODO: Continue replacing parts of this file with components and content from the CMS. -->
 <!-- TODO: Possibly add support for other header variations, e.g. usa-header--basic -->
-<header class="usa-header usa-header--extended">
-  <div class="usa-navbar">
-    <div class="usa-logo" id="basic-logo">
-      <em class="usa-logo__text">
-        <!--TODO: Replace with content from CMS. -->
-        <a href="/" title="<Project title>"> &lt;Project title&gt; </a>
-      </em>
-    </div>
-
+<header class="ldaf-header usa-header usa-header--extended">
```

Comment:
Had to do a quick confidence check on this, myself, but it doesn't affect appearance, since[ the actual rulesets themselves determine precedence](https://css-tricks.com/the-order-of-css-classes-in-html-doesnt-matter/), not class declarations.

I don't have strong feelings on keeping to any particular best practice here, since we're hopefully staying minimal enough on utility classes that the number of any classes on any single element won't get too large

Reply:
Wow, what a trip-up on my part. It's called cascading for a reason 🤦

All good here, disregard!